### PR TITLE
partners: AI21 Labs Contextual Answers support

### PR DESCRIPTION
**Pull request by Josephasafg**
Description: Added support for AI21 Labs model - Contextual Answers
Dependencies: ai21, ai21-tokenizer
Twitter handle: https://github.com/AI21Labs

### CODE REVIEW

**Review by efriis — COMMENTED** [review]
looks great! I'll change some nits in the code, and can you add documentation to the bottom of the `llms/ai21.ipynb` documentation page?

Understand it's not strictly an LLM, and would be great to get some description there!

File: libs/partners/ai21/pyproject.toml [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+version = "0.1.0"
 description = "An integration package connecting AI21 and LangChain"
 authors = []
 readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 langchain-core = "^0.1.22"
-ai21 = "^2.0.0"
+ai21 = "^2.0.5"
```
Comment (efriis) [inline libs/partners/ai21/pyproject.toml:11]: @asafgardin I changed this from `2.0.5` to semver `^2.0.5` which now uses `2.1.2` - were there breaking changes between the two? Lint seems to be saying so

Comment (asafgardin) [reply libs/partners/ai21/pyproject.toml:11]: @efriis Yes. We added better handling for default values that are not passed using `|` so now the lint is throwing an error about it whenever we try to pass `None`. We have a task to better handle these types of values so mypy wouldn't raise an error

**Comment by asafgardin** [comment]
@efriis Added description to `llm/ai21.ipynb`

File: libs/partners/ai21/pyproject.toml (hunk 1 above)
Comment (efriis) [reply libs/partners/ai21/pyproject.toml:11]: Got it. So should I just lock this version to 2.0.5 specifically? Currently causing failures in CI

Comment (efriis) [reply libs/partners/ai21/pyproject.toml:11]: Or do you want to add some logic that programmatically passes the param vs. not, or type it as Any? Happy to follow your lead - you know your sdk better than me and what will work vs. not work :)

Comment (asafgardin) [reply libs/partners/ai21/pyproject.toml:11]: @efriis I'll fix the lint issues next week if that's okay by you. I want to merge this first using version 2.0.5 the way it is right now if everything works in the CI. 

What do you think?

**Review by efriis — APPROVED** [review]

**Pull request merged**